### PR TITLE
Bump `pytorch-optimizer` version to v3.9.0

### PR DESCRIPTION
**Pull request by kozistr**
## Note

related to https://github.com/kozistr/pytorch_optimizer/issues/462,

LoRA training in Kohya_ss could fail when module dropout was enabled together with `pytorch-optimizer` optimizers due to the lazy initialization scheme. This issue has been fixed and released in v3.9.0.

You can also check out the changelog [here](https://github.com/kozistr/pytorch_optimizer/releases)!

Thanks to @JHawkley for bringing this up!

### CODE REVIEW

**Comment by kohya-ss** [comment]
Thank you! This fixes the issue with `module_dropout`!
